index.html

```diff
@@ -0,0 +1,396 @@
+                        <h4>$783.22</h4>
+                    </td>
+                    <td rowspan="2">
+                        <button>
+                            <img src="../img/points.png" alt="">
+                        </button>
+                    </td>
+                    
+                </tr>
+                <tr>
+                    <td>
+                        <a href="https://www.wix.com/">https://www.wix.com/</a>
+                    </td>
+                    <td>
+                        <h4>***** 2468</h4>
+                    </td>
+                    <td>
+                        <a href="mailto:Itaibrahca31@gmail.com">Itaibrahca31@gmail.com</a>
+                    </td>
+                    <td>
+                        <h4>$783.22</h4>
+                    </td>
+                </tr>
+                <tr>
+                    <td rowspan="2">
+                        <input type="checkbox">
+                    </td>
+                    <td rowspan="2">
+                        <img src="../img/youtube.png" alt="">
+                    </td>
+                    <td>
+                        <a href="https://www.youtube.com/">Youtube</a>
+                    </td>
+                    <td>
+                        <h3>Visa</h3>
+                    </td>
+                    <td>
+                        <h3>Itai Bracha</h3>
+                    </td>
+                    <td>
+                        <h3>Jan 2,2022</h3>
+                    </td>
+                    <td rowspan="2">
+                        <img src="../img/done.png" alt="">
+                        <span>Done</span>
+                    </td>
+                    <td rowspan="2">
+                        <h3>Jan 12,2022</h3>
+                    </td>
+                    <td rowspan="2">
+                        <h4>$783.22</h4>
+                    </td>
+                    <td rowspan="2">
+                        <button>
+                            <img src="../img/points.png" alt="">
+                        </button>
+                    </td>
+                    
+                    <tr>
+                        <td>
+                            <a href="https://www.youtube.com/">https://www.youtube.com/</a>
+                        </td>
+                        <td>
+                            <h4>***** 2468</h4>
+                        </td>
+                        <td>
+                            <a href="mailto:Itaibrahca31@gmail.com">Itaibrahca31@gmail.com</a>
+                        </td>
+                        <td>
+                            <h4>$783.22</h4>
+                        </td>
+                    </tr>
+                </tr>
+            </tbody>
+        </table>
+        <div>
+            <div>
+                <button>
+                    <img src="../img/btn_previous.png" alt="">
+                </button>
+                <button>
+                    <img src="../img/btn_1.png" alt="">

```

Comment:
I believe buttons are just a text. The circle around is CSS, thus we don't need images for pagination menu :)

Reply:
Still see images instead of the text